src/components/js/EcApplication.js

```diff
@@ -0,0 +1,86 @@
+import VueMarkdown from 'vue-markdown'
+import EcomApps from '@ecomplus/apps-manager'
+
+export default {
+  name: 'EcApplication',
+  components: {
+    VueMarkdown
+  },
+  data () {
+    return {
+      adminSettings: {},
+      showSettings: false,
+      loading: false,
+      data: null,
+      hiddenData: null
+    }
+  },
+  props: {
+    ecomApps: {
+      type: Object,
+      default: () => new EcomApps()
+    },
+    application: {
+      type: Object,
+      default: () => { }
```

Comment:
você está retornando undefined no default

```suggestion
      default: () => ({})
```